src/core/Entry.cpp

```diff
@@ -220,6 +220,15 @@ QString Entry::notes() const
     return m_attributes->value(EntryAttributes::NotesKey);
 }
 
+QString Entry::lastModified() const
+{
+	// Surely there's a way to ask for sort-by-object and convert to string
+	// later, by giving this item a non-string model. But I'm going to do
+	// a simpler thing for now, to avoid having to learn Qt.
```

Comment:
Perhaps specify the reasoning explicitly:

    the simpler the for now - format the date as one that sorts alphabetically by datetime - to avoid having to learn Qt.